crates/core/component/stake/src/penalty.rs

```diff
@@ -12,52 +10,66 @@ use penumbra_asset::{
     balance::BalanceVar,
     Balance, Value, ValueVar, STAKING_TOKEN_ASSET_ID,
 };
-use penumbra_num::{fixpoint::bit_constrain, Amount, AmountVar};
+use penumbra_num::{
+    fixpoint::{U128x128, U128x128Var},
+    Amount, AmountVar,
+};
 
 /// Tracks slashing penalties applied to a validator in some epoch.
 ///
 /// The penalty is represented as a fixed-point integer in bps^2 (denominator 10^8).
```

Comment:
We should replace this description

Reply:
We can remove all mention of how it is represented internally since the type is opaque